### PR TITLE
Add ?logs response command and anon_reply_without_command config var

### PR DESCRIPTION
**Pull request by papiersnipper**

### CODE REVIEW

**Comment by Taaku18** [comment]
Your branch is behind the development branch, there's currently merge conflicts, to continue, either fix the conflicts or recommit your changes onto a patch branch of development.

File: cogs/modmail.py [hunk 1]
```diff
@@ -704,6 +704,31 @@ def format_log_embeds(self, logs, avatar_url):
...
+        user = user if user is not None else ctx.author
+
+        entries = await self.bot.api.get_responded_logs(user.id)
+
+        embeds = self.format_log_embeds(entries, avatar_url=self.bot.guild.icon_url)
+
+        if not embeds:
+            embed = discord.Embed(
+                color=discord.Color.red(),
+                description="No log entries have been found for that query",
```
Comment (Taaku18) [inline cogs/modmail.py]:
```suggestion
                description="{getattr(user, 'mention', user.id)} has not responded to any thread.",
```

File: core/config.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ class ConfigManager:
         "account_age": None,
         "guild_age": None,
         "reply_without_command": False,
+        "anon_reply_without_command": False,
```
Comment (Taaku18) [inline core/config.py]: Where's your code for this addition?

Comment (Taaku18) [reply core/config.py]: Whoops I missed a section nvm me

File: bot.py [hunk 3]
```diff
@@ -829,7 +829,25 @@ def command_perm(self, command_name: str) -> PermissionLevel:
...
+                        self.config["anon_reply_without_command"]
+                    )
+                except ValueError:
+                    anon_reply_without_command = self.config.remove(
+                        "anon_reply_without_command"
+                    )
+
+                if anon_reply_without_command:
+                    await thread.reply(message, anonymous=True)
+                elif reply_without_command:
```
Comment (Taaku18) [inline bot.py]: The syntax changed for config parsing, as you can see to the [left](https://github.com/kyb3r/modmail/pull/388/files#diff-f289368d1ae6d2d58bec9f799832d932L832), just do:
    if self.config.get("reply_without_command"):
        await thread.reply(message)
    elif self.config.get("anon_reply_without_command"):
        await thread.reply(message, anonymous=True)
    else:
        ...

**Comment by StephenDaDev** [comment]
This closes #316

**Pull request merged**